client/src/pages/ProfileList/List.tsx

```diff
@@ -0,0 +1,134 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+import DatePicker from '../../components/DateRangePicker/DatePicker';
+import SearchIcon from '@material-ui/icons/Search';
+import CalendarTodayIcon from '@material-ui/icons/CalendarToday';
+
+import moment from 'moment';
+import {
+  Grid,
+  Paper,
+  Typography,
+  Box,
+  Card,
+  TextField,
+  Button,
+  CardActions,
+  CardContent,
+  Divider,
+  CardMedia,
+  CardActionArea,
+  Avatar,
+  Input,
+} from '@material-ui/core';
+
+const profileImg = '775db5e79c5294846949f1f55059b53317f51e30.png';
+
+export default function List(): JSX.Element {
+  const classes = useStyles();
+  const [state, setState] = useState({
```

Comment:
```suggestion
  const [state, setState] = useState<Profile[]>([])
```

We can use the Profile interface if we have one. Also, using an array will avoid the need for the variable on line 39